armi/reactor/tests/test_blocks.py

```diff
@@ -1766,6 +1735,126 @@ def test_getReactionRates(self):
         )
 
 
+class BlockRotateTests(unittest.TestCase):
+    """Tests for the ability for a block to be rotated."""
+
+    BOUNDARY_PARAMS = [
+        "cornerFastFlux",
+        "pointsCornerDpa",
+        "pointsCornerDpaRate",
+        "pointsCornerFastFluxFr",
+        "pointsEdgeDpa",
+        "pointsEdgeDpaRate",
+        "pointsEdgeFastFluxFr",
+    ]
+    BOUNDARY_DATA = np.arange(6, dtype=float) * 10
```

Comment:
Does the rotate method work on things that aren't numpy arrays? Like a standard python list? We probably should work on both data types and test that. We had an error early on where Mongoose was adding the data to the TH parameters as numpy objects and that caused errors.

Reply:
I don't see how that test is related to the boundary data rotation.

Reply:
My fault. While the comment is on the boundary data attribute, the call out to "rotate method" focused me elsewhere. I'll modify this test to rotate some boundary params as lists

Reply:
How does 8b81b3e2da7d4920eb52844bcf91a5c854a0d8be look?

Reply:
I'm fine with the content of the test. Honestly, it took me a minute to parse what you were doing in `_assignParamData`. It might be worth elaborating on the comment to be like even index parameters in the the names list get an array and odd index parameters in the names list get a list. Also it would help to rename `referenceData` to `referenceArrayData` and `listData` to `referenceListData`.